pages/1.13/installing/production/system-requirements/zookeeper-requirements/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: layout.pug
+title: ZooKeeper resources
+navigationTitle: ZooKeeper resources
+menuWeight: 10
+excerpt: Requirements and recommendations for ZooKeeper in a DC/OS cluster
+---
+ZooKeeper is a centralized coordination service that stores, maintains, and synchronizes information for distributed systems. ZooKeeper and its management service maintain state information and record details of node activity in a data directory. As changes are made to the cluster, those changes are recorded in the ZooKeeper transaction log. When the transaction log grows too large, ZooKeeper creates a snapshot of the current state of cluster nodes.
+
+# Why planning ZooKeeper resources is important
+As a fundamental component of the DC/OS platform architecture, ZooKeeper performs several critical tasks for the DC/OS cluster. For example, ZooKeeper identifies which master node is used as the leader and coordinates the leader selection so that this information is available to the other master nodes, agents, and schedulers. ZooKeeper also maintains state information for Marathon-orchestrated services and persists this information in its transaction logs and snapshots.
```

Comment:
"and persists this information in its transaction logs and snapshots" seems out of place here. This describes how ZooKeeper persists data, but it is in no way specific to Marathon state. Can we just remove it, or make it an introductory sentence for the "Transaction logs and snapshots" subsection? e.g. "ZooKeeper persists data in its transaction logs and snapshots."

Reply:
Moved under the transaction log /snapshot heading.